flink-ml-lib/src/test/java/org/apache/flink/ml/feature/MinMaxScalerTest.java

```diff
@@ -0,0 +1,208 @@
+        assertEquals(0.0, minMaxScaler.getMin(), 0.0001);
+        assertEquals("prediction", minMaxScaler.getPredictionCol());
+        minMaxScaler
+                .setFeaturesCol("test_features")
+                .setMax(4.0)
+                .setMin(1.0)
+                .setPredictionCol("test_output");
+        assertEquals("test_features", minMaxScaler.getFeaturesCol());
+        assertEquals(1.0, minMaxScaler.getMin(), 0.0001);
+        assertEquals(4.0, minMaxScaler.getMax(), 0.0001);
+        assertEquals("test_output", minMaxScaler.getPredictionCol());
+    }
+
+    @Test
+    public void testFeaturePredictionParam() {
+        MinMaxScaler minMaxScaler =
+                new MinMaxScaler()
+                        .setMin(1.0)
+                        .setMax(4.0)
+                        .setFeaturesCol("test_features")
+                        .setPredictionCol("test_output");
+        MinMaxScalerModel model = minMaxScaler.fit(trainDataTable.as("test_features"));
+        Table output = model.transform(predictDataTable.as("test_features"))[0];
+        assertEquals(
+                Arrays.asList("test_features", "test_output"),
+                output.getResolvedSchema().getColumnNames());
+    }
+
+    @Test
+    public void testFewerDistinctPointsThanCluster() throws Exception {
+        MinMaxScaler minMaxScaler = new MinMaxScaler();
+        MinMaxScalerModel model = minMaxScaler.fit(predictDataTable);
+        Table result = model.transform(predictDataTable)[0];
+        verifyPredictionResult(result, minMaxScaler.getPredictionCol(), Vectors.dense(0.5, 0.5));
+    }
+
+    @Test
+    public void testFitAndPredict() throws Exception {
+        MinMaxScaler minMaxScaler = new MinMaxScaler();
+        MinMaxScalerModel minMaxScalerModel = minMaxScaler.fit(trainDataTable);
+        Table output = minMaxScalerModel.transform(predictDataTable)[0];
+        verifyPredictionResult(output, minMaxScaler.getPredictionCol(), Vectors.dense(0.75, 0.3));
+    }
+
+    @Test
+    public void testSaveLoadAndPredict() throws Exception {
+        MinMaxScaler minMaxScaler = new MinMaxScaler();
+        MinMaxScaler loadedMinMaxScaler =
+                StageTestUtils.saveAndReload(
+                        env, minMaxScaler, tempFolder.newFolder().getAbsolutePath());
+        MinMaxScalerModel minMaxScalerModel = loadedMinMaxScaler.fit(trainDataTable);
+        minMaxScalerModel =
+                StageTestUtils.saveAndReload(
+                        env, minMaxScalerModel, tempFolder.newFolder().getAbsolutePath());
+        assertEquals(
+                Arrays.asList("minVector", "maxVector"),
+                minMaxScalerModel.getModelData()[0].getResolvedSchema().getColumnNames());
+        Table output = minMaxScalerModel.transform(predictDataTable)[0];
+        verifyPredictionResult(output, minMaxScaler.getPredictionCol(), Vectors.dense(0.75, 0.3));
+    }
+
+    @Test
+    public void testModelSaveLoadAndPredict() throws Exception {
```

Comment:
It seems that all functionalities covered in this test is already covered by `testSaveLoadAndPredict`. Could we remove this test for simplicity?

Reply:
OK